packages/components/src/__future__/Tag/Tag/_docs/TagMigration.stories.tsx

```diff
@@ -0,0 +1,321 @@
+
+/** * This is a stand-in component for the legacy Tag's bake in LiveIcon - we should consider adding this as an actual component or replacing it */
+const LiveIconComponent = (): JSX.Element => (
+  <span className={LegacyStyles.liveIcon}>
+    <LiveIcon
+      role="presentation"
+      classNameOverride={LegacyStyles.liveIcon_base}
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+    />
+    <LiveIcon
+      role="presentation"
+      classNameOverride={LegacyStyles.liveIcon_1}
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+    />
+    <LiveIcon
+      role="presentation"
+      classNameOverride={LegacyStyles.liveIcon_2}
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+    />
+    <LiveIcon
+      role="presentation"
+      classNameOverride={LegacyStyles.liveIcon_3}
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+    />
+  </span>
+)
+
+export const LiveIconComponentStory: StoryObj = {
+  render: () => <LiveIconComponent />,
+  parameters: {
+    docs: {
+      source: {
+        type: "dynamic",
+        code: `
+  // component with styled with CSS modules
+  const LiveIconComponent = (): JSX.Element => (
+    <span className={LegacyStyles.liveIcon}>
+      <LiveIcon
+        role="presentation"
+        classNameOverride={LegacyStyles.liveIcon_base}
+        width="16"
+        height="16"
+        viewBox="0 0 16 16"
+        fill="none"
+      />
+      <LiveIcon
+        role="presentation"
+        classNameOverride={LegacyStyles.liveIcon_1}
+        width="16"
+        height="16"
+        viewBox="0 0 16 16"
+        fill="none"
+      />
+      <LiveIcon
+        role="presentation"
+        classNameOverride={LegacyStyles.liveIcon_2}
+        width="16"
+        height="16"
+        viewBox="0 0 16 16"
+        fill="none"
+      />
+      <LiveIcon
+        role="presentation"
+        classNameOverride={LegacyStyles.liveIcon_3}
+        width="16"
+        height="16"
+        viewBox="0 0 16 16"
+        fill="none"
+      />
+    </span>
+  )
+
+  // Minified SCSS from the stylesheet
+  <style>
+    .liveIcon_2,.liveIcon_3{animation-duration:3s;animation-iteration-count:3;animation-delay:1s}.liveIcon{display:inline-block;position:relative;width:20px;height:20px;color:$color-green-500}.liveIcon_1,.liveIcon_2,.liveIcon_3{display:block;position:absolute;top:0;left:$0;width:100%;height:100%;overflow:hidden}.liveIcon_base{opacity:30%;display:block}.liveIcon_1{clip-path:circle(16%)}.liveIcon_2{clip-path:circle(32%);animation-name:pulse-inner}.liveIcon_3{clip-path:circle(50%);animation-name:pulse-outer}@keyframes pulse-inner{0%,25%{opacity:0%}100%,50%,75%{opacity:100%}}@keyframes pulse-outer{0%,25%,50%{opacity:0%}100%,75%{opacity:100%}}
+  </style>
+        `,
+      },
+    },
+  },
+}
+
+export const StatusMigration: StoryObj = {
+  render: () => (
+    <div className="flex gap-12">
```

Comment:
Should use a decorator for this container 😉

Reply:
good call out, there's a few I can add that in for 👍